Category3D.gd

```diff
@@ -23,3 +23,8 @@ func clear_all():
 	self.subcategories = []
 	for child in self.get_children():
 		child.queue_free()
+
+func remove_icon(icon_index: int):
```

Comment:
Seems strange that category3d now handles deleting icons when it does not handle creating them